fluent-package/yum/amazonlinux-2023/Dockerfile

```
@@ -28,6 +28,7 @@ RUN \
   yum install -y ${quiet} yum-utils && \
   yum groupinstall -y ${quiet} "Development Tools" && \
   yum install -y ${quiet} \
+    which \
```

```suggestion
    # install which for building librdkafka 1.9.0 or later 
    which \
```

How about adding a comment like the others?

https://github.com/fluent/fluent-package-builder/blob/dc81219b7174e0b8e1523643df383a1971fc286b/fluent-package/yum/amazonlinux-2/Dockerfile#L48-L49